Formula/presto.rb

```diff
@@ -12,7 +12,7 @@ class Presto < Formula
   resource "presto-cli" do
     version cli_version
     url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-cli/#{version}/presto-cli-#{version}-executable.jar"
```

Comment:
Please remove `#{version}` from the `url`, this is not allowed as per Homebrew style. Explicitly state the version in the URL.